mods/default/nodes.lua

```diff
@@ -1986,6 +1986,45 @@ minetest.register_node("default:sand_with_kelp", {
 -- Corals
 --
 
+local function coral_on_place(itemstack, placer, pointed_thing)
+	if pointed_thing.type ~= "node" or not placer then
+		return itemstack
+	end
+
+	local player_name = placer:get_player_name()
+	local pos_under = pointed_thing.under
+	local pos_above = pointed_thing.above
+	local node_under = minetest.get_node(pos_under)
+	local def_under = minetest.registered_nodes[node_under.name]
+
+	if def_under and def_under.on_rightclick then
```

Comment:
```suggestion
	if def_under and def_under.on_rightclick and not placer:get_player_control().sneak  then
```
You should never be able to trigger an `on_rightclick` callback with sneak.

Reply:
Done.